Mathlib/Topology/Connected/PathConnected.lean

```diff
@@ -969,10 +970,25 @@ theorem IsPathConnected.image' {Y : Type*} [TopologicalSpace Y] (hF : IsPathConn
   refine ⟨(hx y_in).somePath.map' ?_, fun t ↦ ⟨_, (hx y_in).somePath_mem t, rfl⟩⟩
   exact hf.mono (range_subset_iff.2 (hx y_in).somePath_mem)
 
+/-- If `f` is continuous and `F` is path-connected, so is `f(F)`. -/
 theorem IsPathConnected.image {Y : Type*} [TopologicalSpace Y] (hF : IsPathConnected F) {f : X → Y}
     (hf : Continuous f) : IsPathConnected (f '' F) := hF.image' hf.continuousOn
 #align is_path_connected.image IsPathConnected.image
 
+/-- If `h : X → Y` is a homeomorphism, `h(s)` is path-connected iff `s` is. -/
+-- this is not in `Homeomorph.lean` to avoid import cycles
```

Comment:
```suggestion
```
I don't think this is a controversial statement, so I'd remove this line. In fact, I would argue that `Homeomorph.isConnected_image` should be in the connectedness file, but that's not worth refactoring.

Reply:
Sure, removed.